server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/core/Capability.scala

```diff
@@ -167,6 +167,9 @@ case object SubmissionCapabilityFactory extends CapabilityFactory {
   override def id(): CapabilityIdentifier = EMAIL_SUBMISSION
 
   override def create(urlPrefixes: UrlPrefixes): Capability = SubmissionCapability()
+
+   def create(maxDelayedSend: MaxDelayedSend, submissionExtensions: Map[EhloName, EhloArgs]): Capability =
+    SubmissionCapability(EMAIL_SUBMISSION, SubmissionProperties(maxDelayedSend, submissionExtensions))
```

Comment:
TODO: advertise FUTURERELEASE extension in session route + test?

Reply:
No we need to advertise it ONLY if it is supported 

(we do not want to support it on top of RabbitMQ, and an admin might want to turn this off...)

Reply:
ok then another TODO: jmap configuration for this